### DISCUSSION TITLE
Osu! lazer Open GL Configuration Error

### DISCUSSION THREAD

**Discussion by lAg007** [ppy/osu#24529] [Q&A] [answered]
Report: I was messing with the settings at my leisure, and accidentally in the game processing configuration or something like that, which was in auto or Direct X, without knowing I switched to Open GL (legacy) and restarted the program, after that, soon When I started the game I got the following error:

osu!.exe - Sistem Error
Exception Processing Message 0xc0000005 - Unexpected parameters

I've looked into this before, but I've only found solutions where this has happened for other reasons. I would just like to change the game settings through the files or something. I already tried to reinstall the game, but it continues with the error.

System Specs:
Processor: Intel Pentium E5700
GPU: Integrated
RAM: 2GB

**Comment by bdach** [comment]
To make the game work again you can either delete `framework.ini` from the game files, or if feeling particularly attached to your settings, remove the line starting with `Renderer =` from that file.

**Reply by lAg007** [reply, marked as answer]
hi @bdach I didn't find the framework.ini file could you tell me where I can find it?

edit: i find it, is in appdata/roaming/osu Thank you!
